pyndustri.pyi

```diff
@@ -362,6 +362,16 @@ class Unit(Senseable):
         Note that horizon must be in movement for it to shoot.
         """
     @staticmethod
+    def target(self, x: int = None, y: int = None, shoot: bool = False):  # target
+        """
+        Command the unit to target (rotate to) the specified coordinates. If shoot is set to True, it will behave same as shoot(x,y)
+        """
+    @staticmethod
+    def target_prediction(self, unit, shoot: bool = True):  # targetp
```

Comment:
In mindustry assembly, it's 2 separate commands, so I think it should be separate here as well.
Python doesn't have type checking, so it might be hard for users to know what to expect when they are calling function, and playing around with different default calling conventions in signle function would break intuitiveness of it

Reply:
As long as the naming makes sense, it would be fine to have separate commands under the same name (if it was clear and unambiguous). But yeah that's not the case here though so we'll just have to stick with a separate name too.